res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1244,29 +1240,15 @@ NumarkMixtrack3.BrowseButton = function(channel, control, value, status, group)
     var expand;
     var contract;
 
-    if (DarkMetalSkin) {
-        LibraryGroup = "[Hifi]";
-        LibraryCommand = "show";
-        expand = 0;
-        contract = 1;
-    } else {
-        LibraryGroup = "[Master]";
-        LibraryCommand = "maximize_library";
-        expand = 1;
-        contract = 0;
-    }
-
     if (shifted && value === ON) {
```

Comment:
Just curious, you are not keeping the maximize library on the browse button push? In this code sniplet, I don't see any value set for LibraryGroup and Library command nor do I see the actual change in the code to remove the variable